Cargo.toml

```diff
@@ -19,6 +19,7 @@ time = { version = "0.3", features = ["parsing", "formatting"] }
 thiserror = "1.0"
 http = "0.2.5"
 rand = "0.8.4"
+serde = { version = "1.0.140", features = ["derive"] }
```

Comment:
It should be behind a (non-default) feature flag. And then we can add warning in the docs that it serialises to a string.

Reply:
Done.